### PR TITLE
assign null value to unused attribute for shunt creation

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java [hunk 1]
```diff
@@ -140,11 +136,11 @@ public void testCreateToAssignSectionCount() throws Exception {
         mockMvc.perform(post(getNetworkModificationUri()).content(modificationToCreateJson).contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk()).andReturn();
         ShuntCompensatorCreationInfos createdModification = (ShuntCompensatorCreationInfos) modificationRepository.getModifications(getGroupId(), false, true).get(0);
-        assertEquals(0, createdModification.getCurrentNumberOfSections());
+        assertEquals(null, createdModification.getCurrentNumberOfSections());
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java:139]: use assertNull

File: src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java [hunk 2]
```diff
@@ -140,11 +136,11 @@ public void testCreateToAssignSectionCount() throws Exception {
...
         modificationToCreateJson = mapper.writeValueAsString(dto);
         mockMvc.perform(post(getNetworkModificationUri()).content(modificationToCreateJson).contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk()).andReturn();
         createdModification = (ShuntCompensatorCreationInfos) modificationRepository.getModifications(getGroupId(), false, true).get(0);
-        assertEquals(0, createdModification.getCurrentNumberOfSections());
+        assertEquals(null, createdModification.getCurrentNumberOfSections());
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java:144]: use assertNull

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java]: done

File: src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java (hunk 2 above)
Comment (Walid-Sahnoun) [reply src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorCreationInNodeBreakerTest.java]: done

**Review by ayolab — APPROVED** [review]

**Pull request merged**